### PR TITLE
Make chains namespace configurable

### PR DESCRIPTION
**Pull request by piyush-garg**
This makes chains namespace configurable with new flag
chains-namespace where you can specify in which namespace
the chains is configured, by default it is tekton-chains

Do refactoring in chains cmd code

Update docs

Bump chains and hub deps

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [x] Run the code checkers with `make check`
- [x] Regenerate the manpages, docs and go formatting with `make generated`
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

```release-note
Make chains namespace configurable
```

### CODE REVIEW

File: docs/cmd/tkn_chain_signature.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+## tkn chain signature
+
+Print a Tekton chains' signature for a specific taskrun
```
Comment (concaf) [inline docs/cmd/tkn_chain_signature.md:3]:
```suggestion
Print Tekton Chains' signature for a specific taskrun
```

File: docs/cmd/tkn_chain_signature.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+### Usage
+
+```
+tkn chain signature
+```
+
+### Synopsis
+
+Print a Tekton chains' signature for a specific taskrun
```
Comment (concaf) [inline docs/cmd/tkn_chain_signature.md:13]:
```suggestion
Print a Tekton Chains' signature for a specific taskrun
```

File: pkg/cmd/chain/payload.go [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+	SkipVerify bool
+}
+
+func payloadCommand(p cli.Params) *cobra.Command {
+	opts := &PayloadOptions{}
+	f := cliopts.NewPrintFlags("chains")
+
+	c := &cobra.Command{
+		Use:   "payload",
+		Short: "Print a Tekton chains' payload for a specific taskrun",
```
Comment (concaf) [inline pkg/cmd/chain/payload.go:40]:
```suggestion
		Short: "Print Tekton Chains' payload for a specific taskrun",
```

File: pkg/version/version.go [hunk 4]
```diff
@@ -33,14 +33,14 @@ const (
 	oldTriggersControllerSelector  string = "app.kubernetes.io/component=controller,app.kubernetes.io/name=tekton-triggers"
 	dashboardControllerSelector    string = "app.kubernetes.io/part-of=tekton-dashboard,app.kubernetes.io/component=dashboard,app.kubernetes.io/name=dashboard"
 	oldDashboardControllerSelector string = "app=tekton-dashboard"
-	chainsInfo                     string = "chains-info"
+	chainsInfo                     string = "chain-info"
```
Comment (concaf) [inline pkg/version/version.go:36]: keep this as it is?

**Review by concaf — CHANGES_REQUESTED** [review]

File: pkg/version/version.go (hunk 4 above)
Comment (piyush-garg) [reply pkg/version/version.go]: fixed

File: pkg/cmd/chain/payload.go (hunk 3 above)
Comment (piyush-garg) [reply pkg/cmd/chain/payload.go]: fixed

File: docs/cmd/tkn_chain_signature.md (hunk 2 above)
Comment (piyush-garg) [reply docs/cmd/tkn_chain_signature.md]: fixed

File: docs/cmd/tkn_chain_signature.md (hunk 1 above)
Comment (piyush-garg) [reply docs/cmd/tkn_chain_signature.md]: fixed

**Review by vdemeester — APPROVED** [review]

**Review by pradeepitm12 — APPROVED** [review]
/lgtm

**Pull request merged**